library/general/src/Makefile.am

```diff
@@ -93,7 +93,13 @@ ylib2_DATA = \
   lib/ui/text_helpers.rb \
   lib/ui/widgets.rb
 
-EXTRA_DIST = $(module_DATA) $(client_DATA) $(scrconf_DATA) $(agent_SCRIPTS) $(ydata_DATA) $(fillup_DATA) $(ylib_DATA) $(ylib2_DATA)
+ylib3dir = "${yast2dir}/lib/yast2"
```

Comment:
NP: I do not like the numbers in the identifier much, `ylib2dir`, `ylib3dir`,... that's not much descriptive and is a bit confusing.

I'd prefer names like `ylibyast2dir`, `ylibinstallationdir`, `ylibuidir` if possible. (Or maybe with an underscore like `ylib_yast2dir` or `ylib_yast2_dir`, but I'm not sure what automake actually accepts here...)